lib/util/constants.dart

```diff
@@ -2,10 +2,15 @@ import 'package:flutter/material.dart';
 
 final double mainHeight = 62.0;
 
+// Summary
 final Color searchBarBgColor = Color(0xfff2f2f2);
 final Color searchBarPrimaryColor = Color(0xFF828282);
 
 final Color totalCasesBgColor = Color(0xff56CCF2).withOpacity(0.85);
 final Color activeCasesBgColor = Color(0xffF2C94C);
 final Color recoveredBgColor = Color(0xff6FCF97).withOpacity(0.65);
-final Color diedBgColor = Color(0xffBDBDBD).withOpacity(0.80);
\ No newline at end of file
+final Color diedBgColor = Color(0xffBDBDBD).withOpacity(0.80);
+
+final double summarySubFontSize = 20;
+final double summaryMainFontSize = 52;
```

Comment:
Great practice! It is really good to put literal values in variables and put them in a separate file.